app/models/ext_management_system.rb

```diff
@@ -114,6 +114,8 @@ def self.provider_create_params
 
   validates :name,     :presence => true, :uniqueness => {:scope => [:tenant_id]}
   validates :hostname, :presence => true, :if => :hostname_required?
+  validates :zone,     :presence => true
```

Comment:
@agrare @carbonin @Fryguy does this validation make sense in the podified world?  or is it optional there?

Reply:
Yes, it still makes sense. An ems still needs a zone.